### PR TITLE
List Assessments View, and Sync with Esri

### PR DESCRIPTION
**Pull request by rryan0088**
Creation of a List Assessment view, to show an admin user all assessments.

- Allows for filtering by project manager, project and status with the DataTables Javascript library.
- Can update data on the page, from source with a sync button.
- Writes newly synced values to the DB.
- Links to a static "Summary" page, to show the current stages of an assessment.  Still WIP, and put in as a navigation placeholder.
- Includes a feature toggle.  Can provide mock seed data, or live data depending on whether the "UseSeedData" appsettings value is toggled.

Co-authored-by: Jerry Zajac [32240008+jarekzaj@users.noreply.github.com](mailto:32240008+jarekzaj@users.noreply.github.com)
Co-authored-by: Jonny T [90678699+Jonnythan-t@users.noreply.github.com](mailto:90678699+Jonnythan-t@users.noreply.github.com)
Co-authored-by: Ben Hall [3179852+benbhall@users.noreply.github.com](mailto:3179852+benbhall@users.noreply.github.com)
Co-authored-by: Lorna B [93874097+lornacbirchall@users.noreply.github.com](mailto:93874097+lornacbirchall@users.noreply.github.com)
Co-authored-by: Rob Ryan [112186767+rryan0088@users.noreply.github.com](mailto:112186767+rryan0088@users.noreply.github.com)
Co-authored-by: Sathesh Manic [108527825+satheshmanic@users.noreply.github.com](mailto:108527825+satheshmanic@users.noreply.github.com)

### CODE REVIEW

File: src/Elsa.Server/appsettings.Development .json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (jarekzaj) [inline src/Elsa.Server/appsettings.Development .json:1]: we removed this file from source control, so it shouldn't be there as 'new file' - hopefully after this it won't cause any more issues with the file disappearing

File: src/He.PipelineAssessment.Infrastructure/Config/AssessmentConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            builder.Property(x => x.ProjectManagerEmail)
+                .HasMaxLength(100);
+
+            builder.Property(x => x.Status)
+                .HasMaxLength(50);
+
+            if (_useSeedData)
+            {
+                //Do Stuff here.
+                builder.HasData(_dataGenerator.GetAssessments());
```
Comment (jarekzaj) [inline src/He.PipelineAssessment.Infrastructure/Config/AssessmentConfiguration.cs:45]: if I remember, this didn't work, so can probably be removed?

File: src/He.PipelineAssessment.Infrastructure/Data/StubDataSeeder.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using He.PipelineAssessment.Infrastructure.Repository;
```
Comment (jarekzaj) [inline src/He.PipelineAssessment.Infrastructure/Data/StubDataSeeder.cs:1]: also, is this still being used? or now it's just a sync button which goes directly to the repo

**Review by jarekzaj — CHANGES_REQUESTED** [review]
made a few comments, mainly questions, but appsettings.Development change needs to go

File: src/He.PipelineAssessment.Infrastructure/Data/StubDataSeeder.cs (hunk 3 above)
Comment (rryan0088) [reply src/He.PipelineAssessment.Infrastructure/Data/StubDataSeeder.cs]: No longer used - forgot I'd left it in, but I've removed now.

File: src/He.PipelineAssessment.Infrastructure/Config/AssessmentConfiguration.cs (hunk 2 above)
Comment (rryan0088) [reply src/He.PipelineAssessment.Infrastructure/Config/AssessmentConfiguration.cs]: No longer used - forgot I'd left it in, but I've removed now.

**Review by jarekzaj — APPROVED** [review]

**Pull request merged**